<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="1" tests="6" time="61.348">
 <testsuite name="Test Suite" timestamp="2021-03-21T16:17:45" hostname="localhost" time="61.348" errors="0" tests="6" skipped="0" disabled="0" failures="1">
  <testcase classname="Test Suite" name="Successfull register for new users" time="11.789" />
  <testcase classname="Test Suite" name="Login success" time="7.465" />
  <testcase classname="Test Suite" name="Login unsuccessful" time="9.898">
   <failure type="toEqual" message="Expected &apos;http://localhost:4200/login&apos; to equal &apos;http://localhost:4200/user/Qaz&apos;."><![CDATA[Error: Failed expectation
    at C:\Users\hal19\Desktop\Programming\githubtest\testing\angularStudy\angularFirst\projectInsurance\Test\Test.Spec.js:71:39
    at C:\Users\hal19\AppData\Roaming\npm\node_modules\protractor\built\element.js:804:32
    at ManagedPromise.invokeCallback_ (C:\Users\hal19\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\hal19\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\hal19\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at C:\Users\hal19\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
    at C:\Users\hal19\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at processTicksAndRejections (internal/process/task_queues.js:97:5)]]>
   </failure>
  </testcase>
  <testcase classname="Test Suite" name="Insurance registration" time="13.724" />
  <testcase classname="Test Suite" name="Login as guest" time="6.085" />
  <testcase classname="Test Suite" name="Login as guest and apply insurance" time="12.383" />
 </testsuite>
</testsuites>